Makefile.frankfurt

```diff
@@ -58,6 +58,9 @@ DOCKER_REGISTRY = 974517877189.dkr.ecr.eu-central-1.amazonaws.com
 AWS_REGION_ECR := eu-central-1
 AUTHOR=$(USER)
 
+# data.geo.admin.ch hosts
+DATAGEOADMINHOST ?= data.geo.admin.ch
```

Comment:
I think as default we should use DEV staging for local work ?

Reply:
Just for my understanding, what has this env variable to do with the mako and this fix ?

Reply:
right now there is only one data.geo.admin.ch - once, this will be replaced with stac ...

Reply:
The images showed in the tooltip are located on data.geo.admin.ch (and right now only there):
https://mf-chsdi3.prod.bgdi.ch/rest/services/ech/MapServer/ch.babs.kulturgueter/9980/extendedHtmlPopup?lang=en

Reply:
:-) and the csv being filtered by featureID as well on data.geo.admin.ch